dom-manipulation/main.js

```diff
@@ -0,0 +1,26 @@
+const btn = document.querySelector('.hot-button');
+
+let counter = 0;
+
+const mainBtn = btn;
```

Comment:
What's the point of this variable? Why not just use `btn`? This isn't accomplishing anything